digitalidentity/service.go

```diff
@@ -292,3 +279,36 @@ func GetShareReceipt(httpClient requests.HttpClient, receiptId string, clientSdk
 		Error: receiptResponse.Error,
 	}, nil
 }
+
+func validateReceiptResponse(receiptResponse ReceiptResponse) error {
```

Comment:
since you already added the `decryptReceiptContent` function that will protect us from any panics so we don't need this anymore, we can delete this function - it also looks like from a quick chat with pedro that these might actually be optional so better to not validate this

Reply:
removed